client/src/main/java/org/apache/uniffle/client/impl/ShuffleWriteClientImpl.java

```diff
@@ -350,7 +352,7 @@ public void registerShuffle(
     String msg = "Error happened when registerShuffle with appId[" + appId + "], shuffleId[" + shuffleId
         + "], " + shuffleServerInfo;
     throwExceptionIfNecessary(response, msg);
-    shuffleServerInfoSet.add(shuffleServerInfo);
+    registerShuffleServer(appId, shuffleId, shuffleServerInfo);
```

Comment:
rename to `addShuffleServerInfo` ? The `registerShuffleServer ` in this class looks a RPC method.